Package.swift

```diff
@@ -48,5 +48,5 @@ let package = Package(
         .target(name: "AzureCSTextAnalytics", dependencies: ["AzureCore"], path: "AzureCSTextAnalytics", sources: ["Source"]),
         .target(name: "AzureStorageBlob", dependencies: ["AzureCore", "MSAL"], path: "AzureStorageBlob", sources: ["Source"]),
     ],
-    swiftLanguageVersions: [4.2]
+    swiftLanguageVersions: [4, 5]
```

Comment:
@adrianhall the guidelines do say Swift 5, but I recall a prior meeting where we essentially said Swift 4.2 based on your interviews with customers. In any case, I had to change this because 4.2 wasn't valid.

Reply:
We are forward looking, so I think Swift-5 is a good baseline.

Reply:
@adrianhall that is music to my ears because now I can officially use Result<T> and not have to try and backport that to work in Swift 4!

Reply:
Updated.